### PR TITLE
fix: Make Alert form Save CTA sticky

### PR DESCRIPTION
**Pull request by olerichter00**
This PR resolves [ONYX-502] 

### Description

This PR makes the Alert form's "Save" (and "Delete) CTAs sticky and removes the bottom navigation for the "Create Alert" flow and the "Edit Alert" screen ([Slack Discussion](https://artsy.slack.com/archives/C05EQL4R5N0/p1699547193794289)).

| Before | After |
| --- | --- |
|Simulator Screenshot - iPhone 15 Plus - 2023-11-09 at 17 15 16 | Simulator Screenshot - iPhone 15 Plus - 2023-11-09 at 17 10 13|
| --- | --- |
|Simulator Screenshot - iPhone 15 Plus - 2023-11-09 at 17 14 54 |Simulator Screenshot - iPhone 15 Plus - 2023-11-09 at 17 10 51 |

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Make Alert form "Save" CTA sticky (Create and Edit form) - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-502]: https://artsyproduct.atlassian.net/browse/ONYX-502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Make Alert form "Save" CTA sticky (Create and Edit form) - ole)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 602237694f93b0221295f2e866491d4532518dd7
</p>

**Review by gkartalis — COMMENTED** [review]
Thanks for addressing those no-null-assertions 👏

File: src/app/Scenes/SavedSearchAlert/Components/Form.tsx [hunk 1]
```diff
@@ -277,13 +298,14 @@ export const Form: React.FC<FormProps> = ({
             </Button>
           </>
         )}
+
         {!isEditMode && (
-          <Text variant="sm" color="black60" textAlign="center" my={2}>
+          <Text variant="sm" color="black60" textAlign="center" mt={2}>
```
Comment (dblandin) [inline src/app/Scenes/SavedSearchAlert/Components/Form.tsx:303]: While we're in here, according to [this QA card](https://www.notion.so/artsy/Styles-different-to-design-no-text-box-yet-5453ee018c924474abb51dba99c5bc4d), this font size should be `xm`.

**Review by dblandin — APPROVED** [review]
LGTM! Left one suggestion for a quick drive-by change while we're in here but feel free to defer to another PR if you prefer.

**Pull request merged**